metadata-ingestion/src/datahub/ingestion/source/redshift/query.py

```diff
@@ -994,7 +994,7 @@ def temp_table_ddl_query(start_time: datetime, end_time: datetime) -> str:
                                     query_text,
                                     REGEXP_REPLACE(REGEXP_SUBSTR(REGEXP_REPLACE(query_text,'\\\\n','\\n'), '(CREATE(?:[\\n\\s\\t]+(?:temp|temporary))?(?:[\\n\\s\\t]+)table(?:[\\n\\s\\t]+)[^\\n\\s\\t()-]+)', 0, 1, 'ipe'),'[\\n\\s\\t]+',' ',1,'p') AS create_command,
                                     ROW_NUMBER() OVER (
-                                    PARTITION BY session_id, query_text
+                                    PARTITION BY session_id, query_text, query_id
```

Comment:
do we need to partition by both?

```suggestion
                                    PARTITION BY session_id, query_id
```